cmd/main.go

```diff
@@ -85,12 +85,16 @@ func main() {
 		Relayer:           relayer,
 	}
 
-	apiRouter := router.NewRouter(gateway, config.GetRouterConfig(), logger)
+	apiRouter := router.NewRouter(router.RouterParams{
```

Comment:
I don't think we should have `RouterParams`. Seems like an unnecessary level of indirection / abstraction.

Reply:
Removed.